src/mqtt-broker/src/server/grpc/admin/services.rs

```diff
@@ -376,4 +378,43 @@ impl MqttBrokerAdminService for GrpcAdminServices {
 
         Ok(Response::new(reply))
     }
+
+    async fn mqtt_broker_delete_topic_rewrite_rule(
+        &self,
+        request: Request<DeleteTopicRewriteRuleRequest>,
+    ) -> Result<Response<DeleteTopicRewriteRuleReply>, Status> {
+        let req = request.into_inner();
+        let auth_driver = AuthDriver::new(self.cache_manager.clone(), self.client_pool.clone());
+        let config = broker_mqtt_conf();
+        match auth_driver
+            .delete_topic_rewrite_rule(&config.cluster_name, &req.action, &req.source_topic)
+            .await
+        {
+            Ok(_) => Ok(Response::new(DeleteTopicRewriteRuleReply::default())),
+            Err(e) => Err(Status::cancelled(e.to_string())),
+        }
+    }
+
+    async fn mqtt_broker_create_topic_rewrite_rule(
+        &self,
+        request: Request<CreateTopicRewriteRuleRequest>,
+    ) -> Result<Response<CreateTopicRewriteRuleReply>, Status> {
+        let req = request.into_inner();
+        let auth_driver = AuthDriver::new(self.cache_manager.clone(), self.client_pool.clone());
+        let config = broker_mqtt_conf();
+        let topic_rewrite_rule = MqttTopicRewriteRule {
+            cluster: config.cluster_name.clone(),
+            action: req.action.clone(),
+            source_topic: req.source_topic.clone(),
+            dest_topic: req.dest_topic.clone(),
+            re: req.re.clone(),
+        };
+        match auth_driver
```

Comment:
The theme rewriting configuration does not need to be stored via auth_driver, it can be stored directly in the placement center